dist/t/Makefile

```diff
@@ -14,10 +14,11 @@ test:
 	bundle exec rspec --format documentation
 
 dbclean:
-	cd /srv/www/obs/api/
 	@echo "DISABLE_DATABASE_ENVIRONMENT_CHECK=$(DISABLE_DATABASE_ENVIRONMENT_CHECK) should be =1"
 	@echo "SAFETY_ASSURED=$(SAFETY_ASSURED) should be =1"
-	rake db:drop db:create db:setup RAILS_ENV=production
+	cd /srv/www/obs/api/ &&\
+	bundle install &&\
+	bundle exec rake db:drop db:create db:setup RAILS_ENV=production
```

Comment:
This will not drop changes you made to the backend...

Reply:
added code for cleanup of backend (somewhat dangerous if run on an actual production system)

Reply:
🤷🏿‍♂️